src/App.global.css

```diff
@@ -4,8 +4,12 @@
 
 @import '../node_modules/simplebar-react/dist/simplebar.min.css';
 
+body {
+  @apply dark:bg-black dark:bg-opacity-30 bg-white bg-opacity-50;
+}
```

Comment:
this is related to `vibrancy` setting — we don't want the window to be fully transparent.